i18n/en.pot

```diff
@@ -209,6 +209,9 @@ msgstr ""
 msgid "Cannot find dashboard associated with the campaign"
 msgstr ""
 
+msgid "Generating dashboard"
```

Comment:
What about something like 'Looks like the first time you are accessing the dashboard for this campaign. Generating dashboard. This may take up to a couple of minutes'?

Reply:
Changed!